liberty-maven-plugin/src/it/springboot-3-tests/springboot-3-appsdirectory-server-xml-springboot-node-it/src/main/liberty/config/server.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<server description="new server">
+
+    <featureManager>
+    <!-- tag::servlet[] -->
+        <feature>servlet-6.0</feature>
+    <!-- end::servlet[] -->
+    <!-- tag::springboot[] -->
+        <feature>springBoot-3.0</feature>
+    <!-- end::springboot[] -->
+    </featureManager>
+
+    <!-- tag::httpport[] -->
+    <httpEndpoint id="defaultHttpEndpoint"
+                  host="*"
+                  httpPort="9080"
+                  httpsPort="9443" />
+    <!-- end::httpport[] -->
+
+    <springBootApplication id="guide-spring-boot"
```

Comment:
Why do you also have a `src/test/resources/server.xml` for the same test case?

Reply:
Its taking both and overriding with contents from src/test/resources/server.xml 
have removed main liberty config server.xml to avoid confusion